tests/unit/jvm_import/BUILD

```diff
@@ -0,0 +1,3 @@
+load(":jvm_import_test.bzl", "jvm_import_test_suite")
```

Comment:
New files should have a copyright statement in them that looks like the one that's in other files.  In this case, there doesn't seem to be one, but it did, it would look like this here:

https://github.com/bazelbuild/rules_jvm_external/blob/86df919b7ad6708367082f2e17c0fa952919cb63/defs.bzl#L1-L13

Feel free to leave it out and see what they say.